README.md

```diff
@@ -261,6 +261,8 @@ extraFileMappings:
 1. Creating a symbolic link in the chart directory to the external file, or
 2. using `--set-file` to specify the contents of the file. E.g:
    `helm upgrade --install galaxy cloudve/galaxy -n galaxy --set-file extraFileMappings."/galaxy/server/static/welcome\.html".content=/home/user/data/welcome.html`
+   
+Loading the files from external locations still requires a minimal entry in the `extraFileMappings` for that file, as shown above but with no `content:` part. Otherwise, the setup doesn't know on which pods to apply this.
```

Comment:
Can we add those as extra set properties, to be more inline with this particular example?
```
extraFileMappings."/galaxy/server/static/welcome\.html".applyToWeb=true
```

Reply:
Ahhh, cool, how come I didn't thought of that. Sure, will do.